src/components/Booking/RequestButton/index.js

```diff
@@ -33,8 +34,10 @@ const RequestButton = (props) => {
       />
     ) : (
       <Button
-        disabled={(remainingHolidays <= 0
-          || availableDays < 0)}
+        disabled={eventType !== eventTypes.WFH
+          ? ((remainingHolidays <= 0 || availableDays < 0))
```

Comment:
not sure how I missed this but should only need one set of brackets here
`? (remainingHolidays <= 0 || availableDays < 0)`

Reply:
woops, that was silly. Good spot.